pkg/workceptor/remote_work_test.go

```diff
@@ -6,24 +6,84 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net"
+	"os"
+	"path"
```

Comment:
https://pkg.go.dev/path/filepath for filenames. path is for urls. helps with linting.
```suggestion
	"filepath"
```

Reply:
Done, thank you! It was available under `path/filepath`.